### PR TITLE
qa/suites: add "mon down" log variations to ignorelist

### PR DESCRIPTION
**Pull request by ljflores**
Fixes: https://tracker.ceph.com/issues/64864

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [x] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [x] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: qa/suites/orch/cephadm/smoke/start.yaml [hunk 1]
```diff
@@ -2,6 +2,8 @@ overrides:
   ceph:
     log-ignorelist:
       - MON_DOWN
+      - mons down
+      - mon down
```
Comment (adk3798) [inline qa/suites/orch/cephadm/smoke/start.yaml:6]: FYI, there was another variation of this I had saw when testing ignorelist stuff in the cephadm suite https://github.com/ceph/ceph/pull/55765/commits/521931a1ce04641b5d7478530f90331ca2e3df56
It comes out like
```
cluster [WRN] mon.b (rank 2) addr [v2:172.21.15.104:3300/0,v1:172.21.15.104:6789/0] is down (out of quorum)" in cluster log
```

**Review by adk3798 — APPROVED** [review]

File: qa/suites/orch/cephadm/smoke/start.yaml (hunk 1 above)
Comment (ljflores) [reply qa/suites/orch/cephadm/smoke/start.yaml:6]: Will add this too.

**Comment by ljflores** [comment]
Before:
https://pulpito.ceph.com/yuriw-2024-03-26_14:32:05-rados-wip-yuri8-testing-2024-03-25-1419-distro-default-smithi/

After (pending completed runs):
https://pulpito.ceph.com/lflores-2024-04-01_18:07:25-rados-wip-yuri8-testing-2024-03-25-1419-distro-default-smithi/

**Comment by ljflores** [comment]
Added to a few more relevant areas.

**Comment by ljflores** [comment]
jenkins retest this please

**Comment by ljflores** [comment]
CI failures are unrelated.

**Pull request merged**